services/webinar/api.test.js

```diff
@@ -0,0 +1,85 @@
+// @flow
+
+import { expect } from 'chai';
+import nock from 'nock';
+
+import { apiBase } from './utils';
+import * as api from './api';
+
+function expectHttpRequest(status) {
+  const reqheaders = {
+    accept: 'application/vnd.citrix.g2wapi-v1.1+json',
+    authorization: 'token',
+    'content-type': 'application/json',
```

Comment:
Does 'content-type' need be in single quotes here?

Reply:
Yeah I was looking at this, couldn't find a dissallow rule for kebab case